posthog/hogql/parser.py

```diff
@@ -53,6 +58,27 @@ def parse_program(
     return node
 
 
+def parse_string_template(
+    string: str,
+    placeholders: Optional[dict[str, ast.Expr]] = None,
+    timings: Optional[HogQLTimings] = None,
+    *,
+    backend: Optional[Literal["python", "cpp"]] = None,
+) -> ast.Call:
+    """Parse a full template string without start/end quotes"""
+    if not backend:
+        backend = "cpp"
```

Comment:
Nit: I guess the default should be in the function signature now, was in the body just because of earlier percentage-based rollout